### PR TITLE
Emphasize Ably product in descriptions (pubspec, readme)

### PR DESCRIPTION
**Pull request by ben-xD**
Currently, [ably_fluttter shows up in pub.dev](https://pub.dev/packages?q=ably_flutter) as:
CleanShot 2021-11-25 at 10 39 05

As android and iOS are already shown there as platforms, we don't need to mention it in the description. Instead, we should mention Ably, what the SDK actually does more prominently.

I also update the Readme for a similar improvement.

### CODE REVIEW

**Review by lukasz-szyszkowski — APPROVED** [review]

**Review by lukasz-szyszkowski — APPROVED** [review]

File: ios/Classes/codec/AblyFlutterReader.m [hunk 1]
```diff
@@ -146,26 +146,40 @@ use an explicitly received null from Dart (manifesting this side as a nil id)
...
-    ON_VALUE(^(const id value) {
-        [co initWithClientOptions: o hasAuthCallback: value];
-    }, dictionary, TxClientOptions_hasAuthCallback);
-
-    return co;
+    
+    return  [[AblyFlutterClientOptions alloc]
+             initWithClientOptions:o
+             hasAuthCallback:dictionary[TxClientOptions_hasAuthCallback]];
 };
```
Comment (ikbalkaya) [inline ios/Classes/codec/AblyFlutterReader.m:153]: Are these changes supposed to be part of this PR 🤔

**Review by ikbalkaya — APPROVED** [review]
It looks good. However there was a change (`AblyFlutterReader`) which doesn't seem to be part of this PR. I'd expect such a change to be in another PR

File: ios/Classes/codec/AblyFlutterReader.m (hunk 1 above)
Comment (ben-xD) [reply ios/Classes/codec/AblyFlutterReader.m:153]: Sorry @ikbalkaya, i had prematurely merged https://github.com/ably/ably-flutter/pull/252 into this PR by accident.

File: pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: ably_flutter
-description: A wrapper around Ably's Cocoa and Java client library SDKs, providing iOS and Android support.
-version: 1.2.5
+description: Flutter plugin for Ably, a realtime messaging service that powers synchronized digital experiences in realtime.
+version: 1.2.6
```
Comment (QuintinWillison) [inline pubspec.yaml:3]: Changes to `version` should only be made under a release PR. Also, for what its worth, currently released version is at `1.2.8`.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
I'll need to unpick this one a bit - conflicts which perhaps mean we're seeing stuff in this PR that should not be there.

File: pubspec.yaml (hunk 2 above)
Comment (ikbalkaya) [reply pubspec.yaml]: https://github.com/ably/ably-flutter/pull/261/commits/8af11e6540ec7b05dfd1ec34349e8d46e4e4cd53

Comment (ikbalkaya) [reply pubspec.yaml]: https://github.com/ably/ably-flutter/pull/261/commits/8af11e6540ec7b05dfd1ec34349e8d46e4e4cd53

**Comment by QuintinWillison** [comment]
I think we're going to have to take [this commentary](https://github.com/ably/ably-flutter/pull/252#issue-1061407571) at face value / on trust. Presumably, @lukasz-szyszkowski and @ikbalkaya, you were both happy to accept [the explanation](https://github.com/ably/ably-flutter/pull/261#discussion_r765871108) for the conflated nature of this pull request. I'm going to await green CI and then land this.

(this PR is, alas, at the bottom of quite a big stack of other work which needs to be considered for landing)

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**